src/en/02_quick_start.md

```diff
@@ -16,78 +15,78 @@ chmod +x ./pgmanage-$version.AppImage
+The utilities menu is located at the top right corner. From there, you may access the application settings, view the application version, and general application info.
 
 ![Picture of the utilities menu](./images/utilities.png)
 
-On the primary menu, one can manage connections, switch between active database sessions and access the snippets, which will be discussed later in this documentation.
-
+On the primary menu, one can manage connections, switch between active database sessions, and access the snippets, which will be discussed later in this documentation.
 
 ## Protected Credentials Storage
 
-PgManage stores the sensitive data such as database access credentials and SSH keys encrypted with a Master Password.
-Resetting master password will erase all the protected application data.
+PgManage stores sensitive data, such as database access credentials and SSH keys, encrypted with a Master Password. Resetting the master password will erase all the protected application data.
 
 ## Creating your first DB connection
-Click on the ⚡ icon on the left sidebar, the connection management UI will be shown:  
+
+Click on the ⚡ icon on the left sidebar, the connection management UI will be shown:
+  
 ![Connection Management](./images/connection_mgr.png)  
 
-Connections and Connection Groups are shown on the left, clicking on the left panel items shows the item's view/edit form.  
+Connections and Connection Groups are shown on the left. Clicking on the left panel items shows the item’s view/edit form. Click on `➕ Add → Connection`. Set the connection title and database type; the rest of the form will change depending on the database type selected. Fill in the rest of the database connection properties.
 
-Click the `➕ Add` button, select Connection  
-Set connection title and database type, the rest of the form will change depending on the database type selected.
-Fill in the rest of database connection properties.  
-**Note:** Alternatively the connection string may be used to establish a database connection.
+> **Note:** Alternatively, the connection string may be used to establish a database connection.
 
-There are two special connection types which behave differently:
- - SQLite connection does not need any other settings besides the sqlite3 file path.
- - Terminal connection is not a Database database connection, **requires** SSH tunnel properties to be set.
+There are two special connection types, which behave differently:
 
-**Note:** The password field is optional, if you leave it empty the password prompt will be shown each time when establishing the connection. For Postgres connections, PgManage will also try to retrieve the connection password from *.pgpass* file.
+- **SQLite connections** do not need any other settings besides the sqlite3 file path.
+- **Terminal connections** are not database connections, since they require setting SSH tunnel properties.
```

Comment:
I am not sure if this  is correct. Terminal connection is basically a shell/console session with a remote host. It gives you ability to run usual shell commands on the host you're connected to. Such connections use SSH protocol and therefore require you to set SSH properties for the connections.
On the other hand, there are cases when you connect to the remote database via SSH tunnel, in such cases you also need to set SSH files on the connection form.